client/lib/invites/stores/test/invites-list.js

```diff
@@ -1,5 +1,6 @@
-require( 'lib/react-test-env-setup' )();
-
```

Comment:
I noticed that in many tests we can drop `useFakeDom` usage. We should revisit migrated tests later.